index.js

```diff
@@ -151,18 +167,16 @@ TransportStream.prototype._writev = function _writev(chunks, callback) {
 
     if (errState || !transformed) {
       // eslint-disable-next-line callback-return
-      chunks[i].callback();
+      wrapCallback(i);
```

Comment:
This should be `wrapCallback(i)()`, because this code change replaces the `chunks[i].callback()` function *call*, not the function *reference* `chunks[i].callback` (like the other two instances in this PR).

Reply:
Good catch! This probably didn't come up in the tests because there were no errors. Thanks, I'll fix that up.